code/__HELPERS/unsorted.dm

```diff
@@ -1448,3 +1448,40 @@ B --><-- A
 	if(orbiting)
 		loc = get_turf(orbiting)
 		orbiting = null
+
+
+//Center's an image.

```

Comment:
That is NOT an error, that is the correct form of British English, we simply use two different languages/variants.